README.md

```diff
@@ -38,11 +38,11 @@ Thank you for considering and taking the time to contribute! Before contributing
 
 ### ⚡ 3. Skill set :
 
-1. Knowledge of git & github.
+1. [Knowledge of git & github.](https://www.youtube.com/watch?v=4iDQ7CHHZZM)
```

Comment:
We may not need this since we want to make docs promotion free without any third party videos but only official documentation